dissect/target/plugins/apps/browser/chromium.py

```diff
@@ -69,11 +70,12 @@ def _build_userdirs(self, hist_paths: list[str]) -> list[tuple[UserDetails, Targ
                 users_dirs.append((user_details.user, cur_dir))
         return users_dirs
 
-    def _iter_db(self, filename: str) -> Iterator[SQLite3]:
+    def _iter_db(self, filename: str, *subfolders: Optional[list[str]]) -> Iterator[SQLite3]:
```

Comment:
```suggestion
    def _iter_db(self, filename: str, subdirs: Optional[list[str]] = None) -> Iterator[SQLite3]:
```
I don't think using `*` is the nicest approach here. I feel like an explicit list of sub directories is better suited.

I've also renamed it to use the `dir` name, mostly for consistency with the other methods in this file.